services/core-web/src/components/mine/MineHeader.js

```diff
@@ -55,6 +55,8 @@ const propTypes = {
   userInfo: PropTypes.shape({ preferred_username: PropTypes.string.isRequired }).isRequired,
   exemptionFeeStatusOptionsHash: PropTypes.objectOf(PropTypes.string).isRequired,
   governmentAgencyHash: PropTypes.objectOf(PropTypes.string).isRequired,
+  number_of_mine_employees: CustomPropTypes.mine.isRequired,
```

Comment:
These don't need to be added to the propTypes as they exist within `mine = {}`, they should be added to the mine object in `customPropTypes/mines` though.

Reply:
Thanks Meredith :)